go_expvar/check.py

```diff
@@ -63,10 +63,38 @@ def __init__(self, name, init_config, agentConfig, instances=None):
         AgentCheck.__init__(self, name, init_config, agentConfig, instances)
         self._last_gc_count = defaultdict(int)
 
-    def _get_data(self, url):
-        r = requests.get(url, timeout=10)
-        r.raise_for_status()
-        return r.json()
+    def _get_data(self, url, instance):
+        ssl_params = {
+            'ssl': instance.get('ssl', None),
```

Comment:
`None` is already the value returned by default when `get` doesn't find the key, you can simply write `instance.get('ssl')`